app/javascript/utils/EagerPagination.js

```diff
@@ -0,0 +1,36 @@
+import { slice } from 'lodash'
+
+class EagerPagination {
```

Comment:
@creativecoder This is the eager pagination. It presented a few challenges, but it's working now. You can try it here
https://dahlia-lap-full-pr-33.herokuapp.com/applications. When you get to page 6 it should load the next 100 records. However, this is not caching yet. Meaning that if you go back from page 6 to page 5 it will load the previous 100 items again. Caching seems to be straight forward to implement, but I wanted to talk about it in the stand up.

Reply:
in stand-up, we mentioned that there are limits on how much we can cache. seems like we should establish an idea of how much that is. then we can limit ourselves to cache only that much. @creativecoder what do you think about including this caching in this chore vs making a new chore for it?

Reply:
Caching comes with the import UX related concern of how and when to bust the cache. I'd implement pagination to start and follow up with caching later. We may find out we don't actually need it because most users filter to get what they need on the first page.